src/App.tsx

```diff
@@ -1,8 +1,75 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
+import { GoodsList } from './GoodsList/GoodsList';
+import { Good } from './react-app-env';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const API_URL: string = 'https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json';
 
-export default App;
+const getGoods = () => {
+  return fetch(API_URL).then(res => res.json())
+}
+
+interface State {
+  listOfGoods: Good[]
+}
+
+export class App extends Component<{}, State> {
+  state: State = {
+    listOfGoods: [],
+  }
+
+  loadAllGoods = () => {
+    getGoods().then(goods => {
+      this.setState({
+        listOfGoods: goods,
+      })
+    })
+  }
+
+  loadFirstFiveGood = () => {
+    getGoods().then(goods => {
+      this.setState({
+        listOfGoods: goods.filter((good: {id: number}) => good.id <= 5),
```

Comment:
почему линтер не работает?


```suggestion
        listOfGoods: goods.filter((good: { id: number }) => good.id <= 5),
```

Reply:
Лінтер не жалівся на цю строчку